bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -532,13 +537,17 @@ private void showTargetTag() {
         }
 
         @Override public void actionPerformed(final ActionEvent actionEvent) {
-            TargetEnvironment env = obsComp.getTargetEnvironment();
-            GuideEnvironment ge = env.getGuideEnvironment();
+            final TargetEnvironment env = obsComp.getTargetEnvironment();
+            final GuideEnvironment ge = env.getGuideEnvironment();
+
+            // REL-2715 demands that we allow manual guide groups to be automatically created if only the auto
+            // group is present. This requires some ugly hacking.
+            final Option<IndexedGuideGroup> iggOpt = positionTable.getSelectedGroupOrParentGroup(env)
+                    .filter(igg -> !igg.group().isAutomatic())
+                    .orElse(() -> { return appendNewGroup(obsComp, positionTable); });
```

Comment:
Why do you need the block and return here? Would it be enough to do `.orElse(() -> appendNewGroup(obsComp, positionTable));`?